disnake/webhook/async_.py

```diff
@@ -1435,6 +1442,7 @@ async def send(
         avatar_url: Any = MISSING,
         tts: bool = False,
         ephemeral: bool = False,
+        suppress_embeds: bool = False,
```

Comment:
missing in overloads

Reply:
https://github.com/DisnakeDev/disnake/pull/474/commits/04c1be13a6b4664247e0f63d3ece5f98119e9e37